tutorials/physics/collision_shapes_3d.rst

```diff
@@ -0,0 +1,133 @@
+applies after a certain amount of convex shapes. Due to this, for large and
+complex objects such as a whole level, it's recommend to use concave shapes
+instead.
+
+You can generate one or several convex collision shapes from the editor by
+selecting a MeshInstance and using the **Mesh** menu at the top of the 3D
+viewport. The editor exposes two generation modes:
+
+- **Create Single Convex Collision Sibling** uses the Quickhull algorithm.
+  It creates one CollisionShape node with an automatically generated convex
+  collision shape. Since it only generates a single shape, it provides better
+  performance and is recommended for small objects.
+
+- **Create Multiple Convex Collision Siblings** uses the V-HACD algorithm.
+  It creates several CollisionShape nodes, each with their own convex collision
+  shape. Since it genereates multiple shapes, it is more accurate for concave
+  objects at the cost of performance. For objects with medium complexity, it
+  will likely be faster than using a single concave collision shape.
+
+Concave (trimesh) collision shapes
+----------------------------------
+
+:ref:`Concave collision shapes <class_ConcavePolygonShape>` (also called trimesh
+collision shapes) can take any form, from a few triangles to thousands of
+triangles. Concave shapes are the slowest option, but are also the most accurate
+collision shapes available in Godot. **Concave shapes can only be used within
+StaticBodies.** They will not work with KinematicBodies or RigidBodies unless
+the RigidBody's mode is set to Static.
+
+When not using GridMaps for level design, concave collision shapes are usually
+the best approach for level collision. That said, if your level has small
+details, you will probably want to exclude those from collision for better
+performance and reliablity. To do so, you can build a simplified collision mesh
+in a 3D modeller and have Godot generate a collision shape for it automatically
+(see below).
+
+Note that unlike primitive and convex shapes, a concave collision shape doesn't
+have an actual "volume". Objects can be placed both *outside* of the shape as
+well as *inside*.
+
+You can generate a concave collision shape from the editor by selecting a
+MeshInstance and using the **Mesh** menu at the top of the 3D viewport.
+The editor exposes two generation options:
+
+- **Create Trimesh Static Body** is a convenience option. It will create a
+  StaticBody containing a concave shape matching the mesh's geometry.
+
+- **Create Trimesh Collision Sibling** will create a CollisionShape node
+  containing the a concave shape matching the mesh's geometry.
+
+.. note::
+
+    If you need to make a RigidBody *slide* on a concave collision shape, you
+    may notice that sometimes, the RigidBody will bump upwards. To solve this,
+    open the **Project Settings** then create a new boolean property in the
+    ``physics/3d`` category with the name ``smooth_trimesh_collision``. Once
```

Comment:
You may consider to change it, since this solve the issue: https://github.com/godotengine/godot/pull/42193